proto/src/main/proto/Rss.proto

```diff
@@ -116,6 +116,7 @@ message GetMemoryShuffleDataResponse {
   bytes data = 2;
   StatusCode status = 3;
   string retMsg = 4;
+  bool is_end = 5;
```

Comment:
Maybe you can use this `https://github.com/apache/uniffle/blob/43bfd2012b624b3eb91f473582073784cdcf4072/proto/src/main/proto/Rss.proto#L320`intead of creating a v2 response.

Reply:
good to know this.